umh-core/pkg/communicator/actions/deploy-dataflowcomponent.go

```diff
@@ -618,8 +627,17 @@ func (a *DeployDataflowComponentAction) waitForComponentToBeActive(ctx context.C
 							a.outboundChannel, models.DeployDataFlowComponent)
 						continue
 					}
-					if instance.CurrentState == "active" || instance.CurrentState == "idle" {
-						stateMessage := RemainingPrefixSec(remainingSeconds) + fmt.Sprintf("component successfully activated with state '%s', configuration verified", instance.CurrentState)
+					// Compare current state with the desired state
+					var acceptedStates []string
+					switch a.state {
+					case models.ComponentStateActive:
+						acceptedStates = []string{"active", "idle"}
```

Comment:
maybe not hardcode this, but instead leverage like dataflowcomoponent.OperationalStateActive or something like that

Reply:
maybe even list them all (even though the strings behind it are the same) --> it will make it more maintainable, should we choose the rename a state